packages/scandipwa/src/component/CartItem/CartItem.component.js

```diff
@@ -198,12 +198,14 @@ export class CartItem extends PureComponent {
     }
 
     renderBundleProductOptionValue(value) {
-        const { label, quantity, price } = value;
+        const {
+            label, quantity, price, id
+        } = value;
         const { currency_code: currencyCode } = this.props;
         const formattedPrice = formatPrice(price, currencyCode);
 
         return (
-            <div>
+            <div key={ id + label + quantity }>
```

Comment:
Hey! Had to do it that way because ID its the same for all childs like, i.e. Day, month, year all have same id.

Reply:
yea, but can it be that label and quantity also can be same ?

Reply:
Hmm I think label wont repeat, since date is formed from one of each. Maybe I can leave just label as key and test if it complains.

Reply:
it won't complain, but if label can be not unique, then it will complain on some point. I'm just thinking that it can be just `${id}-${index}` even better in this case.